launchable/utils/session.py

```diff
@@ -20,7 +19,14 @@ def _session_file_path(build_name: str) -> Path:
 
 
 def _get_session_id():
-    id = os.getsid(os.getpid())
+    if sys.platform == "win32":
+        import wmi # type: ignore
+        c = wmi.WMI()
+        wql = "Associators of {{Win32_Process='{}'}} Where Resultclass = Win32_LogonSession Assocclass = Win32_SessionProcess".format(os.getpid())
+        res = c.query(wql)
+        id = res[0].LogonId
+    else:
+        id = os.getsid(os.getpid())
```

Comment:
I realize now that I'm looking around this code that the type of `id` will be either `int` or `str` based on the code path. Where I came from, a statically typed language, I'd be inclined make them a consistent type, I'd think that's preferrable here too, unless you feel otherwise.